easybuild/easyconfigs/n/numba/numba-0.37.0-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'Bundle'
+
+name = 'numba'
+version = '0.37.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://numba.pydata.org/'
+description = """Numba is an Open Source NumPy-aware optimizing compiler for Python sponsored by Continuum Analytics,
+ Inc. It uses the remarkable LLVM compiler infrastructure to compile Python syntax to machine code."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+exts_defaultclass = 'PythonPackage'
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
```

Comment:
You can replace this whole block with just `use_pip = True` when using `PythonBundle`